core/src/test/java/io/questdb/griffin/engine/table/AsyncFilteredRecordCursorFactoryTest.java

```diff
@@ -668,50 +762,57 @@ public boolean getCloneSymbolTables() {
+    @Test
+    public void testDeferredSymbolInFilter2TwoPools() throws Exception {
+        withDoublePool((engine, compiler, sqlExecutionContext) -> testDeferredSymbolInFilter0(compiler, sqlExecutionContext));
+    }
 
-                assertCursor(
-                        "s\tt\n",
-                        factory,
-                        true,
-                        true,
-                        false,
-                        false,
-                        sqlExecutionContext
-                );
+    private void testDeferredSymbolInFilter0(SqlCompiler compiler, SqlExecutionContext sqlExecutionContext) throws SqlException {
+        // JIT compiler doesn't support IN operator for symbols.
+        sqlExecutionContext.setJitMode(SqlJitMode.JIT_MODE_DISABLED);
+        compiler.compile("create table x as (select rnd_symbol('A','B') s, timestamp_sequence(20000000, 100000) t from long_sequence(500000)) timestamp(t) partition by hour", sqlExecutionContext);
 
-                // !!! This call is the problematic one !!!
-                compiler.compile("insert into x select rnd_symbol('C','D') s, timestamp_sequence(1000000000, 100000) from long_sequence(100)", sqlExecutionContext);
+        final String sql = "select * from x where s in ('C','D') limit 10";
+        try (final RecordCursorFactory factory = compiler.compile(sql, sqlExecutionContext).getRecordCursorFactory()) {
+            Assert.assertEquals(AsyncFilteredRecordCursorFactory.class, factory.getClass());
 
-                // Verify that all symbol tables (original and views) are refreshed to include the new symbols.
-                assertCursor(
-                        "s\tt\n" +
-                                "C\t1970-01-01T00:16:40.000000Z\n" +
-                                "C\t1970-01-01T00:16:40.100000Z\n" +
-                                "D\t1970-01-01T00:16:40.200000Z\n" +
-                                "C\t1970-01-01T00:16:40.300000Z\n" +
-                                "D\t1970-01-01T00:16:40.400000Z\n" +
-                                "C\t1970-01-01T00:16:40.500000Z\n" +
-                                "D\t1970-01-01T00:16:40.600000Z\n" +
-                                "D\t1970-01-01T00:16:40.700000Z\n" +
-                                "C\t1970-01-01T00:16:40.800000Z\n" +
-                                "D\t1970-01-01T00:16:40.900000Z\n",
-                        factory,
-                        true,
-                        true,
-                        false,
-                        false,
-                        sqlExecutionContext
-                );
-            }
-            resetTaskCapacities();
-        });
+            assertCursor(
+                    "s\tt\n",
+                    factory,
+                    true,
+                    true,
+                    false,
+                    false,
+                    sqlExecutionContext
+            );
+
+            // !!! This call is the problematic one !!!
```

Comment:
This comment looks like a leftover from one of previous PRs.

Reply:
fixed in https://github.com/questdb/questdb/pull/2140/commits/8864a2d55b7aa9268003847388c0277b3d6fab9a